src/Sylius/Bundle/CoreBundle/Resources/config/services/listeners.xml

```diff
@@ -77,6 +77,14 @@
             <argument type="service" id="request_stack" />
             <tag name="kernel.event_listener" event="sylius.order.pre_complete" method="assignCustomerIpToOrder" />
         </service>
+        <service id="sylius.listener.order_complete" class="Sylius\Bundle\CoreBundle\EventListener\OrderCompleteListener">
+            <argument type="service" id="sylius.email_manager.order" />
+            <tag name="kernel.event_listener" event="sylius.order.post_complete" method="sendConfirmationEmail" />
```

Comment:
I guess that we have checkout events with order prefix: http://docs.sylius.org/en/latest/book/orders/checkout.html#checkout-related-events